core/unix/elf_defines.h

```diff
@@ -0,0 +1,237 @@
+            R_386_TLS_DESC /* TLS descriptor containing   \
+                            * pointer to code and to      \
+                            * argument, returning the TLS \
+                            * offset for the symbol.      \
+                            */
+#    endif
+#elif defined(AARCH64)
+#    define ELF_R_TYPE ELF64_R_TYPE
+#    define ELF_R_SYM ELF64_R_SYM
+/* relocation type */
+#    define ELF_R_NONE R_AARCH64_NONE           /* No relocation. */
+#    define ELF_R_DIRECT R_AARCH64_ABS64        /* Direct 64 bit. */
+#    define ELF_R_COPY R_AARCH64_COPY           /* Copy symbol at runtime. */
+#    define ELF_R_GLOB_DAT R_AARCH64_GLOB_DAT   /* Create GOT entry. */
+#    define ELF_R_JUMP_SLOT R_AARCH64_JUMP_SLOT /* Create PLT entry. */
+#    define ELF_R_RELATIVE R_AARCH64_RELATIVE   /* Adjust by program base. */
+#    define ELF_R_IRELATIVE R_AARCH64_IRELATIVE /* STT_GNU_IFUNC relocation. */
+/* tls related */
+#    define ELF_R_TLS_DTPMOD 1028 /* R_AARCH64_TLS_DTPMOD64 Module number. */
+#    define ELF_R_TLS_TPOFF 1030  /* R_AARCH64_TLS_TPREL64  TP-relative offset. */
+#    define ELF_R_TLS_DTPOFF 1029 /* R_AARCH64_TLS_DTPREL64 Module-relative offset. */
+#    define ELF_R_TLS_DESC 1031   /* R_AARCH64_TLSDESC      TLS Descriptor. */
+#elif defined(ARM)
+#    define ELF_R_TYPE ELF32_R_TYPE
+#    define ELF_R_SYM ELF32_R_SYM
+#    define ELF_R_INFO ELF32_R_INFO
+/* relocation type */
+#    define ELF_R_NONE R_ARM_NONE               /* No reloc */
+#    define ELF_R_DIRECT R_ARM_ABS32            /* Direct 32 bit */
+#    define ELF_R_COPY R_ARM_COPY               /* Copy symbol at runtime */
+#    define ELF_R_GLOB_DAT R_ARM_GLOB_DAT       /* GOT entry */
+#    define ELF_R_JUMP_SLOT R_ARM_JUMP_SLOT     /* PLT entry */
+#    define ELF_R_RELATIVE R_ARM_RELATIVE       /* Adjust by program delta */
+#    define ELF_R_IRELATIVE R_ARM_IRELATIVE     /* Adjust indirectly by program base */
+/* tls related */
+#    define ELF_R_TLS_DTPMOD R_ARM_TLS_DTPMOD32 /* Module ID */
+#    define ELF_R_TLS_TPOFF R_ARM_TLS_TPOFF32   /* Negated offsets in static TLS block */
+#    define ELF_R_TLS_DTPOFF R_ARM_TLS_DTPOFF32 /* Offset in TLS block */
+#    ifndef ANDROID
+#        define ELF_R_TLS_DESC                            \
+            R_ARM_TLS_DESC /* TLS descriptor containing   \
+                            * pointer to code and to      \
+                            * argument, returning the TLS \
+                            * offset for the symbol.      \
+                            */
+#    endif                 /* ANDROID */
+#elif defined(RISCV64)
+#    define ELF_R_TYPE ELF64_R_TYPE
+#    define ELF_R_SYM ELF64_R_SYM
+/* relocation type */
+#    define ELF_R_NONE R_RISCV_NONE           /* No relocation. */
+#    define ELF_R_DIRECT R_RISCV_64           /* Direct 64 bit. */
+#    define ELF_R_COPY R_RISCV_COPY           /* Copy symbol at runtime. */
+/* FIXME i#3544: GOT and direct 64 bit both use R_RISCV_64. */
```

Comment:
I know this is a copy but we want to avoid `FIXME` (too strong: gives the impression things are broken) so let's change to `XXX`.